builds/jazz-build-module/vault-utility-module.groovy

```diff
@@ -25,28 +25,34 @@ def initialize(config_loader,service_config, base_url, auth_token, env, event_mo
 	utilModule = util_module
 }
 
-def updateCustomServicesSafeDetails(safeName, lambdaArns, credsId) {
+def updateCustomServicesSafeDetails(safeName, lambdaArns, current_roles, credsId) {
```

Comment:
What is `current_roles` here?

Reply:
The roles which is currently using for deploying the lambda

Reply:
nitpick: can we rename current_roles to `currentRoles` in all places? Thanks.

Reply:
Done